nebula/ui/components/inAppAuth/VPNInAppAuthenticationInputs.qml

```diff
@@ -25,13 +25,21 @@ ColumnLayout {
         return _isSignUpOrIn ? passwordInput : textInput
     }
 
+    function disableActiveInput() {
+        return activeInput().enabled = false;
```

Comment:
The `return` is confusing. Do you use the returned value?

Reply:
good point. no. i'll remove